astroquery/esa/jwst/core.py

```diff
@@ -83,6 +83,15 @@ def __init__(self, *, tap_plus_handler=None, data_handler=None, show_messages=Tr
         if show_messages:
             self.get_status_messages()
 
+    def __check_list_strings(self, list):
+        if list is None:
+            return False
+        if list and all(isinstance(elem, str) for elem in list):
```

Comment:
a single string is iterable and this will return True, is that the logic you want to do here?